cmd/envtool/envtool.go

```diff
@@ -210,14 +211,111 @@ func runCommand(command string, args []string, stdout io.Writer, logger *zap.Sug
 
 	cmd.Stdout = stdout
 	cmd.Stderr = os.Stderr
-
 	if err := cmd.Run(); err != nil {
 		return fmt.Errorf("%s failed: %s", strings.Join(cmd.Args, " "), err)
 	}
 
 	return nil
 }
 
+// runTest runs test with given arguments.
+func runTest(command string, args []string, totalTest int, logger *zap.SugaredLogger) error {
```

Comment:
I think it should be in tests.go since it is the only place where it is used.

Also, having two names `runTest` and `testsRun` is confusing

Reply:
I'll come up with better name

Reply:
I am using execCommandTest, is that okay?

Reply:
Sounds good!